Index: configure.in
===================================================================
--- configure.in	(wersja 275211)
+++ configure.in	(kopia robocza)
@@ -6,7 +6,7 @@
 dnl Don't forget to update the PACKAGE, VERSION and BUILDINFO macros in
 dnl dmake/win95/microsft/config.h to keep the native Microsoft C++
 dnl compiler build happy. 
-AC_INIT(dmake, 4.12)
+AC_INIT(dmake, 4.12-cvs-20090821-PCZERKAS)
 AC_CANONICAL_BUILD
 BUILDINFO="$build"
 
@@ -76,7 +76,7 @@
 	  OS_VERSION=cygwin
 	  CASE_INSENSITIVE_FS=true
 	  ;;
-	"MINGW")
+	MINGW*)
 	  OS_VERSION=mingw
 	  OS_TYPE=winnt
 	  BUILDINFO="Windows / MinGW"
@@ -135,6 +135,12 @@
 AC_DEFINE(CASE_INSENSITIVE_FS, 1, [Assume case insensitive file system.])
 fi
 
+dnl for runing tests under MinGW
+if test x$OS_VERSION = xmingw; then
+    SHELL='sh'
+fi
+
+
 dnl =============================================
 dnl Enable dbug
 dnl =============================================
Index: tests/function_macros-1
===================================================================
--- tests/function_macros-1	(wersja 275211)
+++ tests/function_macros-1	(kopia robocza)
@@ -14,7 +14,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing function macros
-SHELL*:=/bin/sh 
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 TEST1:=a b c
Index: tests/function_macros-10
===================================================================
--- tests/function_macros-10	(wersja 275211)
+++ tests/function_macros-10	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - Check that mktmp can write special characters into a file.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 FOO:=test1\n\ttest2\n\
@@ -43,7 +43,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 2 - Check output of '('
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
@@ -66,7 +66,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 3 - Test multiple line output.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 OBJ = fred.obj mary.obj
@@ -91,7 +91,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 4 - Test <+ ... +>
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 OBJ = fred.obj mary.obj joe.obj
Index: tests/function_macros-2
===================================================================
--- tests/function_macros-2	(wersja 275211)
+++ tests/function_macros-2	(kopia robocza)
@@ -15,7 +15,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing function macros
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 TEST1:=a b c
Index: tests/function_macros-3
===================================================================
--- tests/function_macros-3	(wersja 275211)
+++ tests/function_macros-3	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing continuation char in recipes
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 AA=X\$(shell +echo shellexec)X
Index: tests/function_macros-4
===================================================================
--- tests/function_macros-4	(wersja 275211)
+++ tests/function_macros-4	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing continuation char in recipes
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 aaa :
Index: tests/function_macros-5
===================================================================
--- tests/function_macros-5	(wersja 275211)
+++ tests/function_macros-5	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing continuation char in recipes
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all : all1 all2
Index: tests/function_macros-6
===================================================================
--- tests/function_macros-6	(wersja 275211)
+++ tests/function_macros-6	(kopia robocza)
@@ -15,7 +15,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing nil function macro
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all : dep
Index: tests/function_macros-7
===================================================================
--- tests/function_macros-7	(wersja 275211)
+++ tests/function_macros-7	(kopia robocza)
@@ -15,7 +15,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing mktmp function macro
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
Index: tests/function_macros-8
===================================================================
--- tests/function_macros-8	(wersja 275211)
+++ tests/function_macros-8	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - Test if normpath works.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .WINPATH:=
@@ -36,7 +36,7 @@
 
 # Test 2 - Test if normpath honors .WINPATH setting
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all .WINPATH :
@@ -64,7 +64,7 @@
 
 # Test 3 - Test if the optional parameter works
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
Index: tests/macros-1
===================================================================
--- tests/macros-1	(wersja 275211)
+++ tests/macros-1	(kopia robocza)
@@ -15,7 +15,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing macros
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 TEST1:=a b c
Index: tests/macros-10
===================================================================
--- tests/macros-10	(wersja 275211)
+++ tests/macros-10	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 AAA:=BBB
@@ -39,7 +39,7 @@
 
 # Test 2
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 { := fail
@@ -64,7 +64,7 @@
 
 # Test 3
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 X := :
Index: tests/macros-11
===================================================================
--- tests/macros-11	(wersja 275211)
+++ tests/macros-11	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - Without OOODMAKEMODE set strip leading ./
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 #OOODMAKEMODE=
@@ -41,7 +41,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 2 - With OOODMAKEMODE=y do not strip leading ./
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 OOODMAKEMODE:=y
@@ -65,7 +65,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 3 - Test OOODMAKEMODE from environment.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .IMPORT : OOODMAKEMODE
@@ -89,7 +89,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 4 - With OOODMAKEMODE= leading ./
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 OOODMAKEMODE:=
@@ -113,7 +113,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 5 - Import from environment / targets from command line.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 %target :
Index: tests/macros-2
===================================================================
--- tests/macros-2	(wersja 275211)
+++ tests/macros-2	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all .SETDIR=${TMP} :
@@ -28,7 +28,7 @@
 # Testcase 2
 mkdir -p "$file3"
 cat > $file2 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 
Index: tests/macros-3
===================================================================
--- tests/macros-3	(wersja 275211)
+++ tests/macros-3	(kopia robocza)
@@ -18,7 +18,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1 - No warning expected
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
@@ -34,7 +34,7 @@
 
 # Testcase 2 - dmake should warn
 cat > $file2 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .MYTARGET : something
Index: tests/macros-4
===================================================================
--- tests/macros-4	(wersja 275211)
+++ tests/macros-4	(kopia robocza)
@@ -19,7 +19,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1 - Warn about miising closing braket.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
@@ -28,7 +28,7 @@
 
 # Testcase 2 - warn about $)
 cat > $file2 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
@@ -37,7 +37,7 @@
 
 # Testcase 3 - warn about $:
 cat > $file3 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
@@ -46,7 +46,7 @@
 
 # Testcase 4 - don't warn
 cat > $file4 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
Index: tests/macros-5
===================================================================
--- tests/macros-5	(wersja 275211)
+++ tests/macros-5	(kopia robocza)
@@ -18,7 +18,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
@@ -30,7 +30,7 @@
 mkdir -p "$file3"
 mkdir -p "$file4"
 cat > $file2 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 
Index: tests/macros-6
===================================================================
--- tests/macros-6	(wersja 275211)
+++ tests/macros-6	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .IMPORT .IGNORE : TEMP
Index: tests/macros-7
===================================================================
--- tests/macros-7	(wersja 275211)
+++ tests/macros-7	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - Test if :n works.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .WINPATH:=
@@ -38,7 +38,7 @@
 
 # Test 2 - Test if the .WINPATH setting is honored
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACRO:=/tmp
Index: tests/macros-8
===================================================================
--- tests/macros-8	(wersja 275211)
+++ tests/macros-8	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - Test if :f works.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .WINPATH:=
@@ -38,7 +38,7 @@
 
 # Test 2
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACRO:=/tmp/yyy.ext "/tmp/a  b/xxx.ext"
@@ -60,7 +60,7 @@
 
 # Test 3
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACRO:=/tmp/yyy.ext "/tmp/a  b/xxx.ext"
@@ -82,7 +82,7 @@
 
 # Test 4
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACRO:=/tmp/yyy.ext "/tmp/a  b/xxx.ext"
@@ -104,7 +104,7 @@
 
 # Test 5
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACRO:=/tmp/yyy.ext "/tmp/a  b/xxx.ext"
@@ -126,7 +126,7 @@
 
 # Test 6
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACROd:=/tmp/yyy "/tmp/a  b/xxx"
Index: tests/macros-9
===================================================================
--- tests/macros-9	(wersja 275211)
+++ tests/macros-9	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .WINPATH:=
@@ -38,7 +38,7 @@
 
 # Test 2
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACRO2:=/tmp/zzz.ext /tmp/aaa
@@ -60,7 +60,7 @@
 
 # Test 3
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACRO:="/tmp/yyy.ext" "/tmp/a  b/xxx.ext"
@@ -82,7 +82,7 @@
 
 # Test 4
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACRO:="/tmp/yyy.ext" "/tmp/a  b/xxx.ext"
@@ -106,7 +106,7 @@
 
 # Test 5
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MACROi:="./expand.c"
Index: tests/misc-1
===================================================================
--- tests/misc-1	(wersja 275211)
+++ tests/misc-1	(kopia robocza)
@@ -18,7 +18,7 @@
 # Also quote line continuations ( \<nl> -> \\<nl> ).
 cat > $file1 <<EOT
 # Testing continuation char in recipes
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 .IMPORT : MYMACRO
 all:
Index: tests/misc-10
===================================================================
--- tests/misc-10	(wersja 275211)
+++ tests/misc-10	(kopia robocza)
@@ -15,7 +15,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 MYVAR=
Index: tests/misc-11
===================================================================
--- tests/misc-11	(wersja 275211)
+++ tests/misc-11	(kopia robocza)
@@ -16,7 +16,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 RHS=
Index: tests/misc-12
===================================================================
--- tests/misc-12	(wersja 275211)
+++ tests/misc-12	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles ( $ -> \$ ).
 # Also quote line continuations ( \<nl> -> \\<nl> ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all:
Index: tests/misc-13
===================================================================
--- tests/misc-13	(wersja 275211)
+++ tests/misc-13	(kopia robocza)
@@ -19,7 +19,7 @@
 # Remember to quote variables in generated makefiles ( $ -> \$ ).
 # Also quote line continuations ( \<nl> -> \\<nl> ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .DIRCACHE=yes
Index: tests/misc-14
===================================================================
--- tests/misc-14	(wersja 275211)
+++ tests/misc-14	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles ( $ -> \$ ).
 # Also quote line continuations ( \<nl> -> \\<nl> ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all : 
Index: tests/misc-15
===================================================================
--- tests/misc-15	(wersja 275211)
+++ tests/misc-15	(kopia robocza)
@@ -20,7 +20,7 @@
 # Remember to quote variables in generated makefiles ( $ -> \$ ).
 # Also quote line continuations ( \<nl> -> \\<nl> ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .DIRCACHE=yes
Index: tests/misc-16
===================================================================
--- tests/misc-16	(wersja 275211)
+++ tests/misc-16	(kopia robocza)
@@ -17,7 +17,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all : all1 all2
Index: tests/misc-17
===================================================================
--- tests/misc-17	(wersja 275211)
+++ tests/misc-17	(kopia robocza)
@@ -16,7 +16,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL=/bin/sh
+SHELL=sh
 SHELLFLAGS=-c
 
 my.inc :
Index: tests/misc-18
===================================================================
--- tests/misc-18	(wersja 275211)
+++ tests/misc-18	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - Test if .WINPATH is accepted as global attribute
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .WINPATH :
@@ -38,7 +38,7 @@
 
 # Test 2 - Test if .WINPATH is accepted as local attribute
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all .WINPATH :
@@ -58,7 +58,7 @@
 
 # Test 3 - Test if $@ and $* are DOSified.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 /tmp/blah.xx .WINPATH :
@@ -83,7 +83,7 @@
 
 # Test 4 - Test if $(PWD), $(MAKEDIR) are DOSified.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all .WINPATH .SETDIR=/tmp :
@@ -109,7 +109,7 @@
 
 # Test 5 - Test if $(TMD) is DOSified (if it is an absolute path).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all .WINPATH .SETDIR=/ :
@@ -135,7 +135,7 @@
 
 # Test 6 - Test if .WINPATH changes $@ in conditional macro.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 /tmp/all.bla ?= QQQ=\$@
Index: tests/misc-19
===================================================================
--- tests/misc-19	(wersja 275211)
+++ tests/misc-19	(kopia robocza)
@@ -15,7 +15,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 all :
Index: tests/misc-2
===================================================================
--- tests/misc-2	(wersja 275211)
+++ tests/misc-2	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing continuation char in recipes
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .NOTABS	  !:= yes
Index: tests/misc-20
===================================================================
--- tests/misc-20	(wersja 275211)
+++ tests/misc-20	(kopia robocza)
@@ -20,7 +20,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - from testcase t_81296_5.mk
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 # Test that my.c is build when my.foo is newer.
@@ -68,7 +68,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 2 - from testcase t_81296_6.mk
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 # Test that no circular dependency error is issued:
Index: tests/misc-3
===================================================================
--- tests/misc-3	(wersja 275211)
+++ tests/misc-3	(kopia robocza)
@@ -15,7 +15,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing continuation char in recipes
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 AAA:=aaa
Index: tests/misc-4
===================================================================
--- tests/misc-4	(wersja 275211)
+++ tests/misc-4	(kopia robocza)
@@ -20,7 +20,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing parallel execution
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 testfile : testfile2 testfile3 testfile1
Index: tests/misc-5
===================================================================
--- tests/misc-5	(wersja 275211)
+++ tests/misc-5	(kopia robocza)
@@ -18,7 +18,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Verify that makefile generation is finished when using it
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 incfile.mk : incfiledepend
Index: tests/misc-6
===================================================================
--- tests/misc-6	(wersja 275211)
+++ tests/misc-6	(kopia robocza)
@@ -18,7 +18,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all : allA allB
@@ -34,7 +34,7 @@
 
 # Testcase 2
 cat > $file2 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all  : \$\$@A \$\$@B
Index: tests/misc-7
===================================================================
--- tests/misc-7	(wersja 275211)
+++ tests/misc-7	(kopia robocza)
@@ -16,7 +16,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .SETDIR=subdir : all
Index: tests/misc-8
===================================================================
--- tests/misc-8	(wersja 275211)
+++ tests/misc-8	(kopia robocza)
@@ -18,7 +18,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Verify that makefile generation is finished when using it
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 %.dpcc :
Index: tests/misc-9
===================================================================
--- tests/misc-9	(wersja 275211)
+++ tests/misc-9	(kopia robocza)
@@ -18,7 +18,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 %.dpcc :
Index: tests/recipes-1
===================================================================
--- tests/recipes-1	(wersja 275211)
+++ tests/recipes-1	(kopia robocza)
@@ -15,7 +15,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Testing continuation char in recipes
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all:
Index: tests/recipes-2
===================================================================
--- tests/recipes-2	(wersja 275211)
+++ tests/recipes-2	(kopia robocza)
@@ -15,9 +15,9 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Test group recipes
-SHELL:=/bin/sh 
+SHELL:=sh 
 SHELLFLAGS:=-ce
-GROUPSHELL:=/bin/sh 
+GROUPSHELL:=sh 
 GROUPSUFFIX:=.sh
 
 all :
Index: tests/recipes-3
===================================================================
--- tests/recipes-3	(wersja 275211)
+++ tests/recipes-3	(kopia robocza)
@@ -15,7 +15,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Test noop and empty lines
-SHELL:=/bin/sh 
+SHELL:=sh 
 SHELLFLAGS:=-ce
 
 all :
Index: tests/recipes-4
===================================================================
--- tests/recipes-4	(wersja 275211)
+++ tests/recipes-4	(kopia robocza)
@@ -16,7 +16,7 @@
 cat > $file1 <<EOT
 # internal echo - This test might fail later if the internal echo ever
 # learns to handle quotes.
-SHELL:=/bin/sh 
+SHELL:=sh 
 SHELLFLAGS:=-ce
 
 all :
Index: tests/recipes-5
===================================================================
--- tests/recipes-5	(wersja 275211)
+++ tests/recipes-5	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Test @@ recipe
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 all :
Index: tests/recipes-6
===================================================================
--- tests/recipes-6	(wersja 275211)
+++ tests/recipes-6	(kopia robocza)
@@ -16,9 +16,9 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
 # Test @@ recipe
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
-GROUPSHELL*:=/bin/sh
+GROUPSHELL*:=sh
 GROUPSUFFIX:=.sh
 
 all :
Index: tests/recipes-7
===================================================================
--- tests/recipes-7	(wersja 275211)
+++ tests/recipes-7	(kopia robocza)
@@ -16,7 +16,7 @@
 cat > $file1 <<EOT
 # internal echo - This test might fail later if the internal echo ever
 # learns to handle quotes.
-SHELL:=/bin/sh 
+SHELL:=sh 
 SHELLFLAGS:=-ce
 
 all :
Index: tests/recipes-8
===================================================================
--- tests/recipes-8	(wersja 275211)
+++ tests/recipes-8	(kopia robocza)
@@ -16,9 +16,9 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
-GROUPSHELL*:=/bin/sh 
+GROUPSHELL*:=sh 
 
 all :
 [ This should warn!
@@ -42,9 +42,9 @@
 
 # Test 2
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
-GROUPSHELL*:=/bin/sh 
+GROUPSHELL*:=sh 
 
 all :
 	echo hallo
@@ -69,9 +69,9 @@
 
 # Test 3
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
-GROUPSHELL*:=/bin/sh 
+GROUPSHELL*:=sh 
 
 all :
 [
Index: tests/recipes-9
===================================================================
--- tests/recipes-9	(wersja 275211)
+++ tests/recipes-9	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - Ignore, but display failed execution.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all .IGNORE :
@@ -39,7 +39,7 @@
 
 # Test 2 - Ignore, but display failed execution.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
@@ -62,9 +62,9 @@
 
 # Test 3
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
-GROUPSHELL*:=/bin/sh 
+GROUPSHELL*:=sh 
 
 all :
 	ttttouch all
@@ -85,7 +85,7 @@
 
 # Test 4 - Ignore a succesfully executed, but failing recipe.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
Index: tests/targets-1
===================================================================
--- tests/targets-1	(wersja 275211)
+++ tests/targets-1	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1 - This segfaulted before 4.5
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 allnew : all
@@ -33,7 +33,7 @@
 # Testcase 2 - earlier than dmake 4.5 the second %-target was defined
 # without the prerequisites, i.e. qqq.c was not build
 cat > $file2 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 allnew : qqq.y
Index: tests/targets-10
===================================================================
--- tests/targets-10	(wersja 275211)
+++ tests/targets-10	(kopia robocza)
@@ -18,7 +18,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Check that we warn.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 %.xx : %.yy %.pp ; touch $@
Index: tests/targets-11
===================================================================
--- tests/targets-11	(wersja 275211)
+++ tests/targets-11	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Check that we don't issue an error.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 %.xx .SILENT :
Index: tests/targets-12
===================================================================
--- tests/targets-12	(wersja 275211)
+++ tests/targets-12	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Check that we issue a warning.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .SUFFIXES : .aa .bb
Index: tests/targets-13
===================================================================
--- tests/targets-13	(wersja 275211)
+++ tests/targets-13	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Check that we issue a warning.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all: \$(NEWDEP)
Index: tests/targets-14
===================================================================
--- tests/targets-14	(wersja 275211)
+++ tests/targets-14	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Check that we set AAA to set_AAA.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 .IF \$(AAA)!=321
Index: tests/targets-15
===================================================================
--- tests/targets-15	(wersja 275211)
+++ tests/targets-15	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Check that we set AAA to set_AAA.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 # On cygwin add c:/temp/try5.xx to the prerequisites to test cygwin related
Index: tests/targets-16
===================================================================
--- tests/targets-16	(wersja 275211)
+++ tests/targets-16	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Check that we set AAA to set_AAA.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 DYNPRQS=a\$\$xx1 a\$\$xx2
Index: tests/targets-17
===================================================================
--- tests/targets-17	(wersja 275211)
+++ tests/targets-17	(kopia robocza)
@@ -15,7 +15,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 EMPTYPRQ=
Index: tests/targets-18
===================================================================
--- tests/targets-18	(wersja 275211)
+++ tests/targets-18	(kopia robocza)
@@ -15,7 +15,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 .IMPORT : OOODMAKEMODE
Index: tests/targets-19
===================================================================
--- tests/targets-19	(wersja 275211)
+++ tests/targets-19	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1 - Test if excess leading slashes get removed
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 ////all :
@@ -36,7 +36,7 @@
 
 # Test 2 - Test if excess leading slashes in DOS paths get removed
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 "c:////all" .WINPATH :
Index: tests/targets-2
===================================================================
--- tests/targets-2	(wersja 275211)
+++ tests/targets-2	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1 - %-target prerequisite - syntax error
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .PHONY all : %.c
@@ -27,7 +27,7 @@
 
 # Testcase 2 - The recipe part is ignored - syntax error
 cat > $file2 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all :
Index: tests/targets-20
===================================================================
--- tests/targets-20	(wersja 275211)
+++ tests/targets-20	(kopia robocza)
@@ -26,7 +26,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 %.a : %.b
Index: tests/targets-21
===================================================================
--- tests/targets-21	(wersja 275211)
+++ tests/targets-21	(kopia robocza)
@@ -19,7 +19,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 %.aaa : %.bbb
Index: tests/targets-22
===================================================================
--- tests/targets-22	(wersja 275211)
+++ tests/targets-22	(kopia robocza)
@@ -19,7 +19,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 .REMOVE : ; @echo remove \$<
Index: tests/targets-23
===================================================================
--- tests/targets-23	(wersja 275211)
+++ tests/targets-23	(kopia robocza)
@@ -20,7 +20,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 .REMOVE : ; @echo remove \$<
Index: tests/targets-24
===================================================================
--- tests/targets-24	(wersja 275211)
+++ tests/targets-24	(kopia robocza)
@@ -20,7 +20,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 .REMOVE : ; @echo remove \$<
Index: tests/targets-25
===================================================================
--- tests/targets-25	(wersja 275211)
+++ tests/targets-25	(kopia robocza)
@@ -15,7 +15,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 .REMOVE : ; @echo remove \$<
Index: tests/targets-26
===================================================================
--- tests/targets-26	(wersja 275211)
+++ tests/targets-26	(kopia robocza)
@@ -17,7 +17,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 %.a : %.b
Index: tests/targets-27
===================================================================
--- tests/targets-27	(wersja 275211)
+++ tests/targets-27	(kopia robocza)
@@ -19,7 +19,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 %.x : %.y
Index: tests/targets-28
===================================================================
--- tests/targets-28	(wersja 275211)
+++ tests/targets-28	(kopia robocza)
@@ -19,7 +19,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 1
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 aa.x : aa.y
@@ -52,7 +52,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 2 - Warn if virtual targets have a corresponding file.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 aa.x : aa.y
@@ -89,7 +89,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 3
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 aa.x : aa.y
@@ -128,7 +128,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Test 4
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 aa.x : aa.y
Index: tests/targets-29
===================================================================
--- tests/targets-29	(wersja 275211)
+++ tests/targets-29	(kopia robocza)
@@ -18,7 +18,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 .REMOVE :; @echo remove intermediate target: \$<
Index: tests/targets-3
===================================================================
--- tests/targets-3	(wersja 275211)
+++ tests/targets-3	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1 - Check that warning is issued if .<suffix> targets are ignored
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .nada : something
Index: tests/targets-30
===================================================================
--- tests/targets-30	(wersja 275211)
+++ tests/targets-30	(kopia robocza)
@@ -15,7 +15,7 @@
 
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 OOODMAKEMODE:=y
@@ -35,7 +35,7 @@
 fi
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 cat > $file1 <<EOT
-SHELL*:=/bin/sh
+SHELL*:=sh
 SHELLFLAGS*:=-ce
 
 OOODMAKEMODE:=
Index: tests/targets-4
===================================================================
--- tests/targets-4	(wersja 275211)
+++ tests/targets-4	(kopia robocza)
@@ -19,7 +19,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1 - AUGMAKE meta targets
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .NIX :
@@ -31,7 +31,7 @@
 
 # Testcase 2 - Percent targets
 cat > $file2 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 %.a : %.b
Index: tests/targets-5
===================================================================
--- tests/targets-5	(wersja 275211)
+++ tests/targets-5	(kopia robocza)
@@ -17,7 +17,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Testcase 1 - Check that warning is issued if .<suffix> targets are ignored
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 .NIX.NUX aaa : ; @+echo "Normal and meta target together"
Index: tests/targets-6
===================================================================
--- tests/targets-6	(wersja 275211)
+++ tests/targets-6	(kopia robocza)
@@ -17,7 +17,7 @@
 # Check that .NIX is treated as meta target, making all the target
 # that gets build.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 AUGMAKE=yes
Index: tests/targets-7
===================================================================
--- tests/targets-7	(wersja 275211)
+++ tests/targets-7	(kopia robocza)
@@ -17,7 +17,7 @@
 # Expect dummy, all.pp and all.xx to be build
 # that gets build.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 %.xx %.rr :| %.yy %.pp 'dummy' ; @+printf "\$@"
Index: tests/targets-8
===================================================================
--- tests/targets-8	(wersja 275211)
+++ tests/targets-8	(kopia robocza)
@@ -17,7 +17,7 @@
 # Expect xxx.1 and xxx.2 to be build
 # that gets build.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 all : xxx.1 xxx.2
Index: tests/targets-9
===================================================================
--- tests/targets-9	(wersja 275211)
+++ tests/targets-9	(kopia robocza)
@@ -16,7 +16,7 @@
 # Remember to quote variables in generated makefiles( $ -> \$ ).
 # Check that we warn.
 cat > $file1 <<EOT
-SHELL*:=/bin/sh 
+SHELL*:=sh 
 SHELLFLAGS*:=-ce
 
 %.xx %.rr : %.yy 'a' 'b' ; touch $@
Index: win95/microsft/config.h
===================================================================
--- win95/microsft/config.h	(wersja 275211)
+++ win95/microsft/config.h	(kopia robocza)
@@ -38,7 +38,7 @@
 
 /* Name and version number of this package */
 #define PACKAGE "dmake"
-#define VERSION "4.12"
+#define VERSION "4.12-cvs-20090821-PCZERKAS"
 #define BUILDINFO "Windows / MS Visual C++"
 
 #if defined (_MSC_VER)
Index: winnt/microsft/config.h
===================================================================
--- winnt/microsft/config.h	(wersja 275211)
+++ winnt/microsft/config.h	(kopia robocza)
@@ -27,7 +27,7 @@
 
 /* Name and version number of this package */
 #define PACKAGE "dmake"
-#define VERSION "4.12"
+#define VERSION "4.12-cvs-20090821-PCZERKAS"
 #define BUILDINFO "Windows / MS Visual C++"
 
 #if defined (_MSC_VER)
